frontend/src/metabase/visualizations/components/ChartClickActions/ChartClickActionsView.unit.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { render, screen } from "__support__/ui";
+import DefaultMode from "metabase/modes/components/modes/DefaultMode";
+import { QueryMode } from "metabase-types/types/Visualization";
+import { Column } from "metabase-types/types/Dataset";
+import Mode from "metabase-lib/Mode";
+import { getAdHocQuestion } from "metabase-lib/mocks";
+import ChartClickActionsView from "./ChartClickActionsView";
+
+describe("ChartClickActionsView", () => {
+  it('renders "See this ..." action as the first item', () => {
+    setup();
+
+    const sections = screen.getAllByTestId("drill-through-section");
+
+    expect(sections.length).toBeGreaterThan(1);
+    expect(sections[0]).toHaveTextContent("See these Orders");
+  });
+});
+
+function setup(queryMode: QueryMode = DefaultMode as QueryMode) {
+  const question = getAdHocQuestion();
+  const mode = new Mode(question, queryMode);
+
+  const questionTable = question.table();
+
+  if (!questionTable) {
+    throw new Error("Mock question does not contain a table");
+  }
+
+  const clicked = {
+    column: questionTable.fields[0].column() as Column,
```

Comment:
nit: you could use `checkNotNull` instead of type casting which could be more reliable